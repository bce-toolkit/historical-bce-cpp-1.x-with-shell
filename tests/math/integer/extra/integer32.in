354452151682996010665428244275297910957769595425524696207687606704208815117070980627942707980257549158711789734176493582561151827
771374620572293857036782912694128765412618622680805486020
