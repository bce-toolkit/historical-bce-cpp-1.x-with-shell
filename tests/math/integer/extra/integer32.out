354452151682996010665428244275297910957769595425524696207687606704208815888445601200236565017040461852840555146795116263366637847
354452151682996010665428244275297910957769595425524696207687606704208814345696360055648850943474636464583024321557870901755665807
273415394015504197210209269874057896527113528573281463098341052230179386649287885556061547504891221126952245562866966055894051150039034450655037398805251292561303670524478359951745958540
459507147668434946421090494705889479064273274897558791464995753081370568
196454736250071025288505348678385630600860795427597692467
0
1
0
0
1
1
